include/mbgl/actor/actor_ref.hpp

```diff
@@ -38,7 +38,11 @@ class ActorRef {
     template <typename Fn, class... Args>
     auto ask(Fn fn, Args&&... args) const {
         // Result type is deduced from the function's return type
+#if __cplusplus >= 201703L  // TODO: remove this once fully migrated to C++17
```

Comment:
```suggestion
#if __cplusplus >= 201703L
```

Reply:
I removed the todo and opened an issue instead to keep track of post c++ 17 clean-up tasks. https://github.com/maplibre/maplibre-gl-native/issues/335